### ISSUE TITLE
webpages interlinking

### ISSUE THREAD

**Issue by yashpalgoyal1304** [doublecmd/doublecmd.github.io#52]
Hi!

### Summary:

Uplift the index pages, navigation/header bar, for a more interlinked and complete experience

### Details:

- Other than [gh] & gh-wiki, [mantis tracker], and [forums] ...
- there are mainly 3 official "index" pages of sorts
    + [.com]
    + [.sf.io] (sourceforge)
    + [doc] - hosted in this [.gh.io] repo

The problem is that:
- there's only _ONE_ interlink between them
- i.e. the doc index page links back to the dc.sf.io via this "Homepage" button in the header
- the .sf.io page links to the gh pages in loads, ...
- but NOT to the [doc]
- and the .com pages feels just single isolated page in and off itself.

So, the conclusion is that, these 3 index pages are like _totally isolated islands_

### Solutions:

It would be really nice if:
- the .com and .sf.io ones - i.e. the homepages, could be merged
- the navigation bars of the .sf.io homepage could be tidied up
- and the homepage also links to the doc index page.

[gh]: https://github.com/doublecmd/doublecmd
[.com]: https://doublecommander.com/index.html

[.sf.io]: https://doublecmd.sourceforge.io/index.php
[mantis tracker]: https://doublecmd.sourceforge.io/mantisbt/view_all_bug_page.php
[forums]: https://doublecmd.h1n.ru "Redirected via doublecmd.sourceforge.io/forum/viewtopic.php"

[.gh.io]: https://github.com/doublecmd/doublecmd.github.io/tree/master/doc
[doc]: https://doublecmd.github.io/doc/en/index.html

**Comment by alexx2000** [comment]
- https://doublecommander.com - it is not a official site, I am nothing know about it.
- https://doublecmd.sourceforge.io - official Double Commander site.
- https://doublecmd.github.io/doc/en - official Double Commander documentation.

**Comment by yashpalgoyal1304** [comment]
> * it is not a official site, I am nothing know about it

😱 😨 woah... thats probably the first time i would have fallen victim to an unauthorised website....

but it seems it links back to double commander sourceforge page...

wait, where did i download my dc from :thinking:

**Comment by alexx2000** [comment]
Yes, it links to Double Commander SourceForge project page. I guess this site was created by some enthusiast.

**Comment by yashpalgoyal1304** [comment]
### Screenshots

| Web Search | .com |
| -- | -- |
![search] | ![.com]

[search]: https://user-images.githubusercontent.com/19423063/218318206-4812b022-1eb3-4c89-9783-00ad4742f37c.png
[.com]: https://user-images.githubusercontent.com/19423063/218324086-69a67550-94b6-4010-b914-aa9bfa26ed03.png


|  |  |
| -- | -- |
doc with global-nav | ![doc] 
sf.io sidebar | ![sf-left] 
sf.io nav | ![sf-top]


[doc]: https://user-images.githubusercontent.com/19423063/218325114-de709940-e157-4ea9-90e3-c596c8ff7906.png
[sf-left]: https://user-images.githubusercontent.com/19423063/218325194-c28b2600-add5-4388-8783-bc291960cb01.png
[sf-top]: https://user-images.githubusercontent.com/19423063/218325210-fb0e4e49-5384-424a-be20-1a1b92eed34e.png

**Comment by yashpalgoyal1304** [comment]
just for records: i downloaded from github releases for pc - i _think_ i did same for downloading via other device....

**Comment by Skif-off** [comment]
I don't really understand why there are so many screenshots, but about "Web Search" (pardon my curiosity :)): why?
The official name of the program is "Double Commander", i.e. with space, "doublecmd" is the usual package name in Linux repositories and FreeBSD ports (also "doublecmd-gtl" or "doublecmd-qt"), the abbreviation is also sometimes used (usually "DC"). Sometimes some people write "DoubleCMD".

P.S. I'm not sure, but it seems, it's I was added the "Double Commander Wiki" link: someone is interested in plugins, someone wants to build the program on their own, someone wants to contribute to the program or localization files, but the header is not rubber and just did not want to make it bulky.

**Comment by yashpalgoyal1304** [comment]
> I don't really understand why there are so many screenshots

haha, i just like documenting stuff which seems useful to me. and these screenshots do have purpose - to show at once all the contained links which helps in thinking how they can be better accomodated

as for the comments about the web search - i don't understand what u want to say :sweat_smile: - asking about the missing space?

**Comment by Skif-off** [comment]
> as for the comments about the web search - i don't understand what u want to say sweat_smile - asking about the missing space?

Just a weird (unwonted) result :) Usually in the results I saw the official website, the page on Wikipedia, SF.net/GitHub, but not `https://doublecommander.com` in the first place.
